e2e-app/src/app/tools.po.ts

```diff
@@ -76,7 +46,7 @@ const roundBoundingBox = (rect: {x: number, y: number, width: number, height: nu
  * Returns the element bounding box
  */
 export const getBoundingBox = async(selector: string) => {
-  const element = await test.page.$(selector);
+  const element = await getPage().$(selector);
   const boundingBox = element ? await element.boundingBox() : {x: 0, y: 0, width: 0, height: 0};
   return roundBoundingBox(boundingBox !);
```

Comment:
Can we simplify this code by using locators ? (not directly linked to the test runner, but at the time locators didn't exist).

```typescript
   const boundingBox = (await getPage().locator('selector').boundingBox()) || {x: 0, y: 0, width: 0, height: 0};
```

What do you think ?